tests/rule_based_profiler/data_assistant/test_onboarding_data_assistant.py

```diff
@@ -276,10 +276,10 @@ def test_onboarding_data_assistant_metrics_count(
         domain,
         parameter_values_for_fully_qualified_parameter_names,
     ) in bobby_onboarding_data_assistant_result.metrics_by_domain.items():
-        if domain.is_superset(domain_key):
+        if domain.is_superset(other=domain_key):
             num_metrics += len(parameter_values_for_fully_qualified_parameter_names)
 
-    assert num_metrics == 2
+    assert num_metrics == 4
```

Comment:
Why does this double?

Reply:
@NathanFarmer Because with the suffix now, we have uncovered all the parameter builder names with `raw` in them, which got squashed before -- the suffix fixes this bug now.  Thanks.

Reply:
@alexsherstinsky but everywhere I look I see `suffix=None`, so where is this getting set to affect this test?